src/app/booking/ActivitiesCalendar.js

```diff
@@ -311,7 +311,7 @@ export default ActivitiesCalendar;
 export function ActivityClientBadges({ activity }) {
   return (
     <Stack sx={{ gap: 1 }}>
-      {activity?.goingFast && <Chip label="Going fast" sx={{ bgcolor: "magenta.main", color: "common.white" }} />}
+      {/* {activity?.goingFast && <Chip label="Going fast" sx={{ bgcolor: "magenta.main", color: "common.white" }} />} */}
```

Comment:
What is a business context of a ticket? Why should not badges show up?

Reply:
I think, Sherry doesn't want to show it until there are no real users on the site, but I can clarify it with him

Reply:
You are supposed to understand a business context of a ticket when you start working on it, at the time of PR - especially.